x/stakeibc/keeper/lsm_test.go

```diff
@@ -422,10 +422,11 @@ func (s *KeeperTestSuite) TestDetokenizeLSMDeposit() {
 	s.Require().NotEmpty(delegationICAAddress, "ICA Address should not be empty")
 
 	// Build the host zone and deposit (which are arguments to detokenize)
-	hostZone := types.HostZone{
+	initialHostZone := types.HostZone{
 		ChainId:              HostChainId,
 		DelegationIcaAddress: delegationICAAddress,
 		ConnectionId:         ibctesting.FirstConnectionID,
+		Validators:           []*types.Validator{{DelegationChangesInProgress: 0}},
```

Comment:
I think you get this for free given the field type but never hurts to make sure by setting it explicitly

Reply:
yeah you do, but I think it's good to indicate in the tests which fields are relevant